client/src/components/Lobby.svelte

```diff
@@ -15,6 +15,12 @@
   export let lobby: Lobby;
   let errorMessage: string = "";
 
+  /**
+  variable that will be set true if the corresponding function has no errors thrown
+  this will then allow the button to be pressed again if there is an error thrown
+  */
```

Comment:
each line should start with a `* `

 ```ts
  /**
   * variable that will be set true if the corresponding function has no errors thrown
   * this will then allow the button to be pressed again if there is an error thrown
   */
 ```